src/notation/qml/MuseScore/NotationScene/internal/PercussionPanelToolBar.qml

```diff
@@ -0,0 +1,154 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2024 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+
+import MuseScore.NotationScene 1.0
+
+Item {
+    id: root
+
+    required property var model
+    required property int panelWidth
+
+    property alias navigation: navPanel
+
+    // anchors.fill: parent TODO: un-comment when #22050 is finished
```

Comment:
I don't think we should uncomment it :) We should always set the anchors at a higher level, not inside a component